lib/services/access_list.go

```diff
@@ -297,3 +297,68 @@ func SelectNextReviewDate(accessList *accesslist.AccessList) time.Time {
 
 	return nextDate
 }
+
+// AccessListReviewsGetter defines an interface for reading access list reviews.
+type AccessListReviewsGetter interface {
+	// ListAccessListReviews will list access list reviews for a particular access list.
+	ListAccessListReviews(ctx context.Context, accessList string, pageSize int, pageToken string) (reviews []*accesslist.Review, nextToken string, err error)
+}
+
+// AccessListReviews defines an interface for managing Access List reviews.
+type AccessListReviews interface {
+	AccessListReviewsGetter
+
+	// CreateAccessListReview will create a new review for an access list. It will also modify the original access list
+	// and its members depending on the details of the review.
+	CreateAccessListReview(ctx context.Context, review *accesslist.Review) (reviewName string, nextAuditDate time.Time, err error)
+
+	// DeleteAccessListReview will delete an access list review from the backend.
+	DeleteAccessListReview(ctx context.Context, accessList, name string) error
```

Comment:
I'll make these clearer. This is because reviews are namespaces to a particular access list, so knowing the access list name is essential before deleting one.